### DISCUSSION TITLE
Question about CSP_SM

### DISCUSSION THREAD

**Discussion by narata2019** [NREL/ReEDS-2.0#126] [Q&A: Features]
Hi ReEDS team,

There is a parameter called CSP_SM in the model, which varies with storage durations, and I am not sure about the meaning.
Can I treat it as follows:  If CSP_4 (6h) should equiped with 1 propoation solar resource. Then CSP_1, CSP_2, CSP_3 and CSPNS should equiped with 2.7, 2.4, 1.3 and 1.4 propoartion solar resources.
Is that right? Many thanks!

Siming

**Comment by vcelloho** [comment]
CSP_SM is short for solar multiple. It's the ratio between the maximum thermal power that the solar field can collect and the turbine. CSP with longer duration storage has a higher solar multiple since it makes sense to use the larger collector field to charge the thermal storage during the day and produce a more stable and dispatchable generation output.

**Reply by narata2019** [reply]
Thank you. For a same steam turbine, it seems the non-storage CSP also has a larger collector field, compared with 6h and 8h storage CSP. Why this mean?
